back/app/controllers/web_api/v1/permissions_controller.rb

```diff
@@ -31,8 +31,14 @@ def update
 
   def requirements
     authorize @permission
-    json_requirements = user_requirements_service.requirements @permission, current_user
-    render json: raw_json({ requirements: json_requirements }), status: :ok
+    permissions_service = Permissions::BasePermissionsService.new(current_user, user_requirements_service: user_requirements_service)
+    requirements = user_requirements_service.requirements @permission, current_user
+    json_requirements = {
+      permitted: user_requirements_service.permitted?(requirements),
```

Comment:
If we already have `disabled_reason`, do we still need `permitted`? Wouldn't they contain the same information?

Reply:
Yes kind of. But does make it easier to test that the same outcomes are present as we had before. Suggest we leave it in for now, and remove later